dapp/src/components/page/view/PublicProfileView.svelte

```diff
@@ -51,14 +51,14 @@
     </div>
   {/if}
   <div class="flex flex-row items-center">
-    <div class="flex flex-row items-center cursor-pointer bubble-outline">
-      <div class="address-container">
-        {$params.address}
+    <div class="flex flex-row items-center cursor-pointer bubble-outline mr-0">
+      <div class="address-container mr-2">
+        {$params.address.slice(0,5)}...{$params.address.slice(-4)}
```

Comment:
Second time I'm seeing this function, please consider a component

Reply:
{$userData.account.address.slice(0,5)}...{$userData.account.address.slice(-4)}
----
{$params.address.slice(0,5)}...{$params.address.slice(-4)}

different info for each time, it's not same, maybe better not to create separate component for each one?:)

Reply:
Mmmm, I thought I might've seen that somewhere else, my bad.